src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs

```diff
@@ -184,7 +184,9 @@ internal static ControlFlowGraph CreateCore(IOperation operation, string argumen
                 Debug.Assert(controlFlowGraph.OriginalOperation == operation);
                 return controlFlowGraph;
             }
-            catch (Exception e) when (FatalError.ReportAndCatchUnlessCanceled(e, cancellationToken))
+#pragma warning disable CS0618 // ReportIfNonFatalAndCatchUnlessCanceled is obsolete
+            catch (Exception e) when (FatalError.ReportIfNonFatalAndCatchUnlessCanceled(e, cancellationToken))
+#pragma warning restore CS0618 // ReportIfNonFatalAndCatchUnlessCanceled is obsolete
             {
                 // Log a Non-fatal-watson and then ignore the crash in the attempt of getting flow graph.
                 Debug.Assert(false, "\n" + e.ToString());
```

Comment:
This Debug.Assert prevents the compiler from flagging that the "return null" below is actually disallowed by the nullable annotations.